### ISSUE TITLE
Belt and Disc Sander: Disk Sander Needs Repair!

### ISSUE THREAD

**Issue by reogaro** [comakingspace/do-something#304]
### TL;DR: The disc sanding portion broke. 

Replacement parts are on the way (please donate if you can, info below), only use the belt end for now, if you have experience with fixing/commissioning this machine and/or want to help comment on this issue.

### Full explanation:

The Disc Sander portion of the Belt and Disc Sander has broken, in two ways:
First of all, the disc isn't orientated perpendicular to the axis of rotation. This leads to unclean sanding and would probably have been a somewhat easy fix if it hadn't led to problem 2:
Secondly, the Velcro on the disc end isn't working anymore. The wobbly rotation combined with previous damage has led to a sanding disk ripping and workpiece-velcro contact.

So the machine requires:
- Velcro Replacement
- Disc side alignment

For this, I have already placed an order for [replacement velcro](https://www.holz-metall.info/Klettauflage-fuer-Schleifteller-diverse-Groessen/KLETTAUFLAGE200) along with some 100 grit sandpaper for good measure. With shipping cost this has come out to 23€, if someone has a few bucks to donate to this good cause message me on telegram, I'd be really appreciative, university doesnt pay :(

**For now, only use the belt sanding side until the disc sander is fixed!** If someone has experience working on this machine and has advice, or someone wants to help, or someone with experience wants to help, please comment!

**Comment by reogaro** [comment]
TL:DR Update: I have pulled the disk off. It doesn't need "realignment", it's warped and either needs to be bent/turned/ground flat or replaced. I'm working on it.

Long version, may be useful as documentation for future repairs:

It's not misaligned, but bent and warped and needs major repair or replacement. The manufactuerer has a [replacement part catalogue](https://ersa.holzmann-maschinen.at/de), but doesn't list this particular machine or one with a compatible plate size. Sigh. They do list a service mail address on their [homepage](https://www.holzmann-maschinen.at/DE/) though, maybe annoying them will lead to results.

Grinding and turning it flat would throw off the disks weight distribution and require balancing, which is why I plan on trying to bend it back into shape. It's an aluminium part, so that'll be interesting, it really likes cracking when bent.

On a more positive note, isopropanol was quite effective at neutralizing the adhesive holding on the old velcro. It was hiding a M6x16 Left handed flathead phillips screw that was all chewed up as if someone had tried to loosen it the wrong (not right?) direction. Pun intended. Replacing it with a hex or torx drive would be a good idea, just like all of the annoying phillips head screws all over this machine.

Back to the topic. For actually removing the plate from the motor shaft, I modeled and 3D printed and built a plate puller tool (first attatched picture). I'll see if I can post the files soon, but a physical copy all ready and built can now be found in the machines' drawer. Please don't lose it, along with the little bag of machine parts in there. The two side screws screw into the plate, while the central one threads through a M8 nut to push itself off the motor shaft.

For bending it, I'll try to form it using clamps, maybe. Seems more controllable than a hammer.

![photo_2023-11-23_01-44-11](https://github.com/comakingspace/do-something/assets/34105511/70a4d1bc-9082-46d6-802e-94634980d6bd)
![photo_2023-11-23_01-44-12](https://github.com/comakingspace/do-something/assets/34105511/8287dc81-f1b6-46ae-b76f-6ac9fa1d1a66)

**Comment by reogaro** [comment]
Johannes has taken the disk to his turning contacts and the face was lathed flat, thanks for that! Let's hope that fixed it and hasn't throwin it too far off balance.

We're now only wating for the M6x16 left hand thread repacement screw, then we can put it back together.

**Comment by Johannes09** [comment]
Neues Problem, das Gewinde in der Welle ist kaputt. Ein Linksgewindeschneider zum nachschneiden ist bestellt.

**Comment by Johannes09** [comment]
Repariert 
![20240110_192707](https://github.com/comakingspace/do-something/assets/22060480/ad7c36ed-ac56-468c-847e-f8f4260ab218)
